packages/compartment-mapper/src/import-hook.js

```diff
@@ -751,9 +836,9 @@ export function makeImportNowHookMaker(
         if (exitRecord) {
           // It'd be nice to check the policy before importing it, but we can only throw a policy error if the
           // hook returns something. Otherwise, we need to fall back to the 'cannot find' error below.
-          enforceModulePolicy(moduleSpecifier, compartmentDescriptor, {
+          enforcePolicyByModule(moduleSpecifier, compartmentDescriptor, {
```

Comment:
(should) Verify whether it's intentional that only async importHook flow calls `moduleSourceHook` for the exit modules it finds and here it's not being called.

Also, I'm not sure if there is a point in calling it for exit modules. What would be the usecase?

Reply:
Well, we only use this at time of generation. Generation _only_ calls `importHook` and never `importNowHook`.  So that's why.  

We aren't using this on the LavaMoat side, but I'm not sure why we wouldn't, just semantically speaking.  These are all `ModuleSource` objects whether they represent source on disk, a builtin, or an error.  🤷